src/main/java/backend/resource/TurboLabel.java

```diff
@@ -91,26 +98,109 @@ private void ______METHODS______() {
         }
     }
 
-    private static String joinWith(String group, String name, boolean exclusive) {
-        return group + (exclusive ? EXCLUSIVE_DELIMITER : NONEXCLUSIVE_DELIMITER) + name;
+
+    /**
+     * Assumption: the labelName matches at least 1 TurboLabel
+     * @return the first TurboLabel in labels that matches labelName
+     */
+    public static TurboLabel getFirstMatchingTurboLabel(List<TurboLabel> labels, String labelName) {
+        Optional<TurboLabel> firstMatchingLabel = getMatchedLabels(labels, labelName).stream().findFirst();
+        assert firstMatchingLabel.isPresent();
```

Comment:
We need to. Ideally, they should be enabled for rc version but not for release version. But if that's too hard, we can enable it for all.